src/components/ViewRoutingSlip/RoutingSlipInfo.vue

```diff
@@ -114,18 +114,18 @@
         <v-btn
           large
           color="primary"
-          @click="updateStatus()"
+          @click="updateStatus(isApprovalFlow)"
           class="px-8 font-weight-bold"
           data-test="btn-edit-routing-done"
         >
-          <span>Done</span>
+          <span>{{isApprovalFlow ? 'Authorize' :  'Done'}}</span>
```

Comment:
Do we have to pass isApprovalFlow or can we take from computed property from the composable function itself?